### PR TITLE
✨ script to sort locale key

### LINKED ISSUE #11015: automatically run scripts before making a commit
after #11009

> would be nice if the script automatically ran before making a commit

maybe in new pr?

```ts
  "lint-staged": {
    "*": [
      "pnpm lint:fix",
      "pnpm locales:order"
    ]
  },
```

_Originally posted by @roiLeo in https://github.com/kodadot/nft-gallery/issues/11009#issuecomment-2360818645_

### PR DESCRIPTION
**Pull request by roiLeo**
- related with #4868 
- `pnpm locales:order`
- closes #11015

### CODE REVIEW

File: components/common/party/CursorParty.vue [hunk 1]
```diff
@@ -135,7 +135,7 @@ watch(
       const updateLabel
         = !isEqual(prevConnection?.lastEvent, connection.lastEvent)
         || (prevConnection?.lastEvent
-          && Date.now() - prevConnection.lastEvent.timestamp > 1000)
+        && Date.now() - prevConnection.lastEvent.timestamp > 1000)
```
Comment (roiLeo) [inline components/common/party/CursorParty.vue:138]:
```suggestion
          && Date.now() - prevConnection.lastEvent.timestamp > 1000)
```

File: components/create/Confirm/MintConfirmModal.vue [hunk 2]
```diff
@@ -128,8 +128,8 @@ const kodadotFee = computed(
 const carbonlessFee = computed(
   () =>
     ((preferencesStore.hasCarbonOffset && isNFT.value ? BASE_FEE * 2 : 0)
-      / tokenPrice.value)
-      * Math.pow(10, decimals.value),
+    / tokenPrice.value)
+    * Math.pow(10, decimals.value),
```
Comment (roiLeo) [inline components/create/Confirm/MintConfirmModal.vue:132]:
```suggestion
      / tokenPrice.value)
      * Math.pow(10, decimals.value),
```

File: components/create/CreateNft.vue [hunk 3]
```diff
@@ -572,9 +572,9 @@ watchEffect(() => {
   // prepare nft blockNumber for redirect to detail page
   if (
     (transactionStatus.value === 'mint'
-      || transactionStatus.value === 'list')
-      && mintStatusFinalized
-      && blockNumber.value
+    || transactionStatus.value === 'list')
+    && mintStatusFinalized
+    && blockNumber.value
```
Comment (roiLeo) [inline components/create/CreateNft.vue:577]:
```suggestion
      || transactionStatus.value === 'list')
      && mintStatusFinalized
      && blockNumber.value
```

File: components/profile/activityTab/History.vue [hunk 4]
```diff
@@ -210,8 +210,8 @@ const createTable = (): void => {
         if (previousPriceMap[nftId]) {
           event['Percentage']
             = ((parseInt(newEvent['meta']) - previousPriceMap[nftId])
-              / previousPriceMap[nftId])
-              * 100
+            / previousPriceMap[nftId])
+            * 100
```
Comment (roiLeo) [inline components/profile/activityTab/History.vue:214]:
```suggestion
              / previousPriceMap[nftId])
              * 100
```

File: components/profile/create/stages/Form.vue [hunk 5]
```diff
@@ -245,9 +245,9 @@ const deleteConfirmSafetyDelayText = computed(() => {
     return $i18n.t('profiles.waitSeconds', [
       Math.ceil(
         (deleteConfirm.value.getTime()
-          + DELETE_CONFIRM_SAFETY_DELAY
-          - now.value.getTime())
-          / 1000,
+        + DELETE_CONFIRM_SAFETY_DELAY
+        - now.value.getTime())
+        / 1000,
```
Comment (roiLeo) [inline components/profile/create/stages/Form.vue:250]:
```suggestion
          + DELETE_CONFIRM_SAFETY_DELAY
          - now.value.getTime())
          / 1000,
```

File: components/shared/DynamicGrid.vue [hunk 6]
```diff
@@ -58,7 +58,7 @@ const grid = computed(
   () =>
     props.gridSize
     || (props.gridSection
-      && preferencesStore.getGridConfigBySection(props.gridSection)?.size),
+    && preferencesStore.getGridConfigBySection(props.gridSection)?.size),
```
Comment (roiLeo) [inline components/shared/DynamicGrid.vue:61]:
```suggestion
      && preferencesStore.getGridConfigBySection(props.gridSection)?.size),
```

File: components/shared/TransactionSteps/utils.ts [hunk 7]
```diff
@@ -36,8 +36,8 @@ export const getTransactionStepDetails = (
 
   if (
     (status !== TransactionStatus.Unknown
-      && step.stepStatus !== TransactionStepStatus.WAITING)
-      || step.stepStatus === TransactionStepStatus.LOADING
+    && step.stepStatus !== TransactionStepStatus.WAITING)
+    || step.stepStatus === TransactionStepStatus.LOADING
```
Comment (roiLeo) [inline components/shared/TransactionSteps/utils.ts:40]:
```suggestion
      && step.stepStatus !== TransactionStepStatus.WAITING)
      || step.stepStatus === TransactionStepStatus.LOADING
```

File: composables/autoTeleport/useAutoTeleportTransactionActions.ts [hunk 8]
```diff
@@ -9,7 +9,7 @@ export const getAutoTeleportActionInteraction = (
   autoTeleportAction: AutoTeleportAction,
 ): AutoteleportInteraction =>
   (autoTeleportAction.action?.interaction
-    || autoTeleportAction.interaction) as AutoteleportInteraction
+  || autoTeleportAction.interaction) as AutoteleportInteraction
```
Comment (roiLeo) [inline composables/autoTeleport/useAutoTeleportTransactionActions.ts:12]:
```suggestion
    || autoTeleportAction.interaction) as AutoteleportInteraction
```

File: composables/autoTeleport/useAutoTeleportTransitionDetails.ts [hunk 9]
```diff
@@ -117,8 +117,8 @@ export default function (
   const hasBalances = computed(
     () =>
       (Boolean(currentChainBalance.value)
-        && Object.values(sourceChainsBalances.value).every(Boolean))
-        || hasFetched.balances,
+      && Object.values(sourceChainsBalances.value).every(Boolean))
+      || hasFetched.balances,
```
Comment (roiLeo) [inline composables/autoTeleport/useAutoTeleportTransitionDetails.ts:121]:
```suggestion
        && Object.values(sourceChainsBalances.value).every(Boolean))
        || hasFetched.balances,
```

File: composables/usePrefix.ts [hunk 10]
```diff
@@ -19,9 +19,9 @@ export default function () {
   const prefix = computed<Prefix>(
     () =>
       (sharedPrefix.value
-        || validPrefixFromRoute.value
-        || storage.value.selected
-        || initialPrefixFromPath) as Prefix,
+      || validPrefixFromRoute.value
+      || storage.value.selected
+      || initialPrefixFromPath) as Prefix,
```
Comment (roiLeo) [inline composables/usePrefix.ts:24]:
```suggestion
        || validPrefixFromRoute.value
        || storage.value.selected
        || initialPrefixFromPath) as Prefix,
```

**Review by roiLeo — COMMENTED** [review]
manual linter

**Comment by vikiival** [comment]
Coolio, seems there is a ton of duplicates in the i18n files

File: utils/math.ts [hunk 11]
```diff
@@ -75,8 +75,8 @@ export const between
   = (dateA: Date, dateB: Date) =>
     (event: Interaction): boolean =>
       (isAfter(parseISO(event.timestamp), dateA)
-        || isEqual(parseISO(event.timestamp), dateA))
-        && isBefore(parseISO(event.timestamp), dateB)
+      || isEqual(parseISO(event.timestamp), dateA))
+      && isBefore(parseISO(event.timestamp), dateB)
```
Comment (roiLeo) [inline utils/math.ts:79]:
```suggestion
        || isEqual(parseISO(event.timestamp), dateA))
        && isBefore(parseISO(event.timestamp), dateB)
```

**Comment by hassnian** [comment]
would be cool if the script automatically ran before making a commit

**Comment by roiLeo** [comment]
> would be nice if the script automatically ran before making a commit

maybe in new pr?

```
  "lint-staged": {
    "*": [
      "pnpm lint:fix",
      "pnpm locales:order"
    ]
  },
```

**Review by hassnian — APPROVED** [review]

**Comment by hassnian** [comment]
> > would be nice if the script automatically ran before making a commit
> 
> maybe in new pr?
> 
> ```ts
>   "lint-staged": {
>     "*": [
>       "pnpm lint:fix",
>       "pnpm locales:order"
>     ]
>   },
> ```

 https://github.com/kodadot/nft-gallery/issues/11015 🚀

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
It would be very nice to clear out unused translations

**Pull request merged**